app/models/carrier.rb

```diff
@@ -2,6 +2,8 @@
 
 class Carrier < ApplicationRecord
   belongs_to :location
+  alias_attribute :home_location, :location
```

Comment:
Nice trick with alias here, but I think it would be better to rename location column -> home_location at the moment. Project is still in development phase we don't have any production customers so no need to be afraid of renaming the column :)

Reply:
Good point @dixpac I think in the long run having a name that is not ambiguous can save us pain in the future when working on this and asking ourselves "is location home_location or current_location".

Reply:
Ah I see - makes sense. I've updated the migration now (realised after I could have used a `rename_column` but kept as is since adding as a reference adds it as a bigint instead of integer)